src/Dropdown/Dropdown.js

```diff
@@ -19,23 +19,60 @@ export default class Dropdown extends Util.mixin(BindMixin) {
   constructor() {
     super();
     this.state = {
+      menuDirection: 'down',
+      menuHeight: null,
       isOpen: false,
-      selectedID: null
+      selectedID: null,
+      knowMenuHeight: false
```

Comment:
No, it won't toggle where it's placed if the dropdown is already open and the items change. I meant we could tell it to recalculate its height before it's opened by setting  `knowMenuHeight` to `false`, and we could do this in `componentWillReceiveProps` if its number of items change.